incubation.md

```diff
@@ -0,0 +1,79 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects are created and evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a GitHub repository with a development team that follows the Jupyter governance and contribution model. The goals of the incubation process are:
+
+* Contributors can quickly and easily get their code exposed to the Jupyter community while complying with corporate contribution restrictions.
```

Comment:
As there are more and more newcomers to the community, I think yes. It's hard to go back through years of mailing list posts to find out what exists in the Jupyter ecosystem, especially if it's not under the Jupyter name somewhere. 

Likewise, as one of these newcomers, it's hard for me to tell at a glance what under the github.com/jupyter/* repo is experimental vs mature, active vs dead except to ask about each one or guess based on evidence like time of last commit, amount of activity, docs, etc.